src/main/scala/dpla/ingestion3/PreMappingReporterMain.scala

```diff
@@ -72,20 +42,25 @@ object PreMappingReporterMain {
       """.stripMargin)
   }
 
+  val logger: Logger = LogManager.getLogger("PreMappingReporter")
+
   def main(args: Array[String]): Unit = {
     if (args.length < 4) {
       usage()
       System.err.println("Incorrect invocation arguments")
       sys.exit(1)
     }
-    val inputURI = args(0)
-    val outputURI = args(1)
+    val input = args(0)
+    val output = args(1)
     val sparkMasterName = args(2)
     val token = args(3)
 
-    new PreMappingReporter(
-      token, inputURI, outputURI, sparkMasterName
-    ).main()
+    val result = new PreMappingReporter(input, output, sparkMasterName, token)
```

Comment:
Here in this file the last parameter and its variable are named `token` but elsewhere in `PreMappingReporter` it's referred to as `inputDataType`, which might be more clear, here and above as "input data type" in the usage documentation.